src/Host/Broker/Impl/Microsoft.R.Host.Broker.csproj

```diff
@@ -42,16 +75,38 @@
     <Compile Include="..\..\..\GlobalAssemblyInfo.cs">
       <Link>Properties\GlobalAssemblyInfo.cs</Link>
     </Compile>
+    <Compile Include="..\..\Client\Impl\Definitions\Message.cs">
+      <Link>Protocol\Message.cs</Link>
```

Comment:
Referencing it like this is a hack until we can untangle dependencies. We'll probably need a new shared assembly that contains just the protocol definitions - specifically, Message from Host.Client, and all the *Info structs from Host.Broker. That can then be referenced by both Host.Client and Host.Broker.